pluggable/lazyload/lazyload.php

```diff
@@ -214,6 +214,10 @@ function process_image_attributes( array $attributes ) : array {
 	if ( ! empty( $attributes['class'] ) && should_skip_image_with_blacklisted_class( $attributes['class'] ) ) {
 		return $attributes;
 	}
+	// Exclude custom logo from lazy loading.
+	if ( 'custom-logo' === $attributes['class'] ) {
```

Comment:
I think we should change this to `preg_match( '/\bcustom-logo\b', $attributes['class'] )` to account for multiple classes, as right now it only matches exactly that one class being used.

Please test this to make sure it works, just in case. Then it can be merged IMO.

Reply:
Tested, and works. I'm updating the PR and merging. Thanks @felixarntz!